stylesheets/lib/postcss-minify-selectors.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const selectorParser = require('postcss-selector-parser')
+
+module.exports = (opts) => {
+  return {
+    postcssPlugin: 'postcss-minify-selectors',
+    Rule (rule) {
```

Comment:
I think we should explain this custom rule.
Do we need it in order to generate the exact same default stylesheet? Is it an intermediate step? Do we want to keep it?

Reply:
I agree, the rules need some explanation. The purpose of this rule is two fold:

* Minify the selectors without sorting them
* Don't break pseudo elements (I don't know why cssnano insists on using the incorrect syntax)